kuzzleio/kuzzle-admin-console
# [KZL-1484] Refactor list collection indexes

## What does this PR do ?
Enhances the experience of loading the index and collection list. Now indexes are loaded first (thus rendered), then collections for each index are loaded afterwards. Some glitches still persist in the Treeview but the overall bootstrap process is better.

### How should this be manually tested?
Connect the AC to a Kuzzle with a lot of indexes and collections and see if the bootstrap is smooth.

### Other changes
* Applied `scroll` to document searches like #632

### TODO
- [x] Loader when opening the collection list for an index
